web/templates/pages/members.html

```diff
@@ -16,42 +16,47 @@
 		</div>
 	</div>
 	
-
+	</div>
 	<div class="row">
-		<div>
-			<ul class="about_list">
-
+		<div class="col-lg-6 col-sm-7">
+			<h3>Kittens</h3>
+		</div>
+	</div>
+    
+	<div class="kitten-list">
+			<div class="row">
 			{% for member in members_list %}
-				{% if member.user.is_active %}
-					{% if member.activity_status == 'MENTOR' %}
-						<li class="mentor-cat">
-					{% elif member.activity_status == 'ACTIVE' %}
-						<li class="active-cat">
-					{% elif member.activity_status == 'LAZY' %}
-						<li class="lazy-cat">
-					{% elif member.activity_status == 'SLEEPY' %}
-						<li class="sleepy-cat">
-					{% endif %}	
-
+				<div class="col-md-3 col-xs-6">
+					{% if member.user.is_active and member.activity_status == 'ACTIVE' %}
```

Comment:
You are correct @mojcas. We could display mentors above everybody else
(active lazy and sleepy).
On Feb 27, 2015 1:47 PM, "mojcas" <notifications@github.com> wrote:

> In web/templates/pages/members.html
> <https://github.com/CodeCatz/TrackCat/pull/214#discussion_r25503922>:
>
> >  			{% for member in members_list %}
> > -				{% if member.user.is_active %}
> > -					{% if member.activity_status == 'MENTOR' %}
> > -						<li class="mentor-cat">
> > -					{% elif member.activity_status == 'ACTIVE' %}
> > -						<li class="active-cat">
> > -					{% elif member.activity_status == 'LAZY' %}
> > -						<li class="lazy-cat">
> > -					{% elif member.activity_status == 'SLEEPY' %}
> > -						<li class="sleepy-cat">
> > -					{% endif %}	
> > -
> > +				<div class="col-md-3 col-xs-6">
> > +					{% if member.user.is_active and member.activity_status == 'ACTIVE' %}
>
> Remember we have Active field on User as well, and those who are not
> active, are already filtered out in view, but on UserProfile we have an
> activity status. Here it seems now we are not displaying even Sleepy or
> Lazy Kittens? Why we have those statuses then? :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/CodeCatz/TrackCat/pull/214/files#r25503922>.
>